include/seqan3/alignment/matrix/alignment_matrix_formatter.hpp

```diff
@@ -0,0 +1,383 @@
+     */
+    static const alignment_matrix_format csv;
+
+    /*!\brief A format that uses only ascii symbols.
+     * \sa https://en.wikipedia.org/wiki/Ascii
+     *
+     *
+     * ### Score output
+     * \snippet test/snippet/alignment/matrix/alignment_matrix_formatter.out alignment_matrix_format::ascii#score
+     *
+     * ### Trace output
+     * \snippet test/snippet/alignment/matrix/alignment_matrix_formatter.out alignment_matrix_format::ascii#trace
+     */
+    static const alignment_matrix_format ascii;
+
+    /*!\brief A format that uses unicode block symbols.
+     * \sa https://en.wikipedia.org/wiki/Unicode
+     *
+     *
+     * ### Score output
+     * \snippet test/snippet/alignment/matrix/alignment_matrix_formatter.out alignment_matrix_format::unicode_block#score
+     *
+     * ### Trace output
+     * \snippet test/snippet/alignment/matrix/alignment_matrix_formatter.out alignment_matrix_format::unicode_block#trace
+     */
+    static const alignment_matrix_format unicode_block;
+
+    /*!\brief A format that uses unicode braille symbols.
+     * \sa https://en.wikipedia.org/wiki/Unicode
+     *
+     *
+     * ### Score output
+     * \snippet test/snippet/alignment/matrix/alignment_matrix_formatter.out alignment_matrix_format::unicode_braille#score
+     *
+     * ### Trace output
+     * \snippet test/snippet/alignment/matrix/alignment_matrix_formatter.out alignment_matrix_format::unicode_braille#trace
+     */
+    static const alignment_matrix_format unicode_braille;
+
+   /*!\brief A format that uses unicode arrow symbols.
+    * \sa https://en.wikipedia.org/wiki/Unicode
+    *
+    *
+    * ### Score output
+    * \snippet test/snippet/alignment/matrix/alignment_matrix_formatter.out alignment_matrix_format::unicode_arrows#score
+    *
+    * ### Trace output
+    * \snippet test/snippet/alignment/matrix/alignment_matrix_formatter.out alignment_matrix_format::unicode_arrows#trace
+    */
+    static const alignment_matrix_format unicode_arrows;
+};
+
+constexpr alignment_matrix_format alignment_matrix_format::csv
+{
+    " ", ";", "", "",
+    {"N","D","U","DU","L","DL","UL","DUL"}
+};
+
+constexpr alignment_matrix_format alignment_matrix_format::ascii
+{
+    " ", "|", "-", "/",
+    {" ","D","U","DU","L","DL","UL","DUL"} // 1byte per char
+};
+
+constexpr alignment_matrix_format alignment_matrix_format::unicode_block
+{
+    u8"ε", u8"║", u8"═", u8"╬",
+    {u8"█",u8"▘",u8"▝",u8"▀",u8"▖",u8"▌",u8"▞",u8"▛"}, // 3bytes per char
+    2, 3
+};
+
+constexpr alignment_matrix_format alignment_matrix_format::unicode_braille
+{
+    u8"ε", u8"║", u8"═", u8"╬",
+    {u8"⠀",u8"⠁",u8"⠈",u8"⠉",u8"⠄",u8"⠅",u8"⠌",u8"⠍"}, // 3bytes per char
+    2, 3
+};
+
+constexpr alignment_matrix_format alignment_matrix_format::unicode_arrows
+{
+    u8"ε", u8"║", u8"═", u8"╬",
+    {u8"↺",u8"↖",u8"↑",u8"↖↑",u8"←",u8"↖←",u8"↑←",u8"↖↑←"}, // 3bytes per char
+    2, 3
+};
```

Comment:
This is 2 months old code. Way before the whole workshop hype started and it was incredible helpful for debugging. Furthermore, this will be a cool feature once the debug stream is in.